examples/resources/bloxone_federation_federated_block/resource.tf

```diff
@@ -0,0 +1,16 @@
+resource "bloxone_federation_federated_realm" "example" {
+  name = "example_federation_federated_realm"
+}
+
+resource "bloxone_federation_federated_block" "example" {
```

Comment:
please change the example to be consistent with examples of other resources, especially names and values

Reply:
done